src/harness-manager/values.yaml

```diff
@@ -212,7 +212,7 @@ featureFlags:
   # -- STO Feature Flags
   CD: "CDNG_ENABLED"
   # -- STO Feature Flags
-  CI: "CING_ENABLED,CI_INDIRECT_LOG_UPLOAD"
+  CI: "CING_ENABLED,CI_INDIRECT_LOG_UPLOAD","CI_LE_STATUS_REST_ENABLED"
```

Comment:
could you remove the extra quote, it should be comma seperated onlyCI: "CING_ENABLED,CI_INDIRECT_LOG_UPLOAD,CI_LE_STATUS_REST_ENABLED"